projects/plugins/jetpack/modules/carousel/jetpack-carousel.php

```diff
@@ -320,6 +328,15 @@ function enqueue_assets() {
 			$localize_strings = apply_filters( 'jp_carousel_localize_strings', $localize_strings );
 			wp_localize_script( 'jetpack-carousel', 'jetpackCarouselStrings', $localize_strings );
 			wp_enqueue_style( 'jetpack-carousel', plugins_url( 'jetpack-carousel.css', __FILE__ ), array(), $this->asset_version( JETPACK__VERSION ) );
+			wp_enqueue_style(
+				'jetpack-carousel-swiper-css',
+				Assets::get_file_url_for_environment(
+					'_inc/build/carousel/swiper-bundle.min.css',
```

Comment:
For the bundle to be minified, I believe we'll need to add its reference here:
https://github.com/Automattic/jetpack/blob/e13487d0dbbe4c743739b6f1a3e37c143a8c2614/projects/plugins/jetpack/tools/builder/frontend-css.js#L25
https://github.com/Automattic/jetpack/blob/b1c8b8ecba896ab616a2d7ebe43ecf5d6a6463db/projects/plugins/jetpack/class.jetpack.php#L63

Reply:
Perfect, working out how to get the css into the build process was on my todo list todo, that saves me some hunting! Done.